src/background/tab-manager.ts

```diff
@@ -211,6 +211,8 @@ export default class TabManager {
                     break;
                 }
             }
+            sendResponse('Nothing-to-be-seen-here');
```

Comment:
Maybe worth sending something like {type: 'anything'}? Less chance to break anything, like object destructuring.

Reply:
A lemmy face should do it :)